subsys/bluetooth/host/classic/l2cap_br.c

```diff
@@ -4928,9 +4929,21 @@ static int l2cap_br_recv(struct bt_l2cap_chan *chan, struct net_buf *buf)
 			continue;
 		}
 
+		if (ident == 0) {
```

Comment:
was  it maybe  last ident？ and  if so l2cap_br_send_reject would reject last cmd